core/src/main/java/com/graphhopper/GraphHopper.java

```diff
@@ -521,6 +521,7 @@ public GraphHopper init(CmdArgs args) {
         if (!flagEncodersStr.isEmpty() || !encodedValueStr.isEmpty()) {
             if (!encodedValueStr.isEmpty())
                 emBuilder.addAll(tagParserFactory, encodedValueStr);
+            registerCustomEncodedValues(emBuilder);
```

Comment:
Ah, this is a bit unfortunate that this is necessary but we would need yet another factory (EncodedValueFactory) plus a method in EncodingManager to support this.

Reply:
Yeah, talking about factories, extension points and such is a new issue, or epic. But I don't think it's that bad in this case. "Builder" sounds like something that should be safe to pass around and add stuff to.